src/bioregistry/data/bioregistry.json

```diff
@@ -190,6 +193,14 @@
     "example": "T0599",
     "mappings": {
       "cellosaurus": "ABM"
+    },
+    "name": "Applied Biological Materials cell line products",
+    "synonyms": [
+      "abm"
+    ],
+    "uri_format": "https://www.abmgood.com/Cell-Biology.html$1",
```

Comment:
This doesn't seem to work with an example, what's the motivation for adding this?

Reply:
good afternoon, I only added it so that it was similar to the one I uploaded previously, in that they told me that it was good and I tried to make it as similar as possible, but if you prefer I will remove it.

Reply:
The point of the URI format string is to make it possible to resolve a given local identifier - this is not the same as a homepage (perhaps there was some confusion here - https://cthoyt.com/2021/10/07/biopragmatics-glossary.html provides some more background). In a URI format string, the `$1` is a placeholder for the local unique identifier. You can see that there's already one of these available via Cellosaurus for this record.